src/ergw_aaa_gx.erl

```diff
@@ -300,13 +301,13 @@ handle_cca('CCA', RC, AVPs, Session0, Events0, _Opts, State)
     {ok, Session, Events, State};
```

Comment:
missed one in the line directly above this one:

```
 {Session, Events} = to_session({gy, 'CCA'}, {Session0, Events0}, AVPs),
```

the `gy` in there is clearly wrong, that should be `?API`

Reply:
Fixed. Thanks.